Makefile

```diff
@@ -44,16 +44,25 @@ CFLAGS += $(DEBUGFLAGS) $(MOREFLAGS)
 FLAGS   = $(CFLAGS) $(CPPFLAGS)
 XXHSUM_VERSION = $(LIBVER)
 
+ifeq '$(findstring ;,$(PATH))' ';'
```

Comment:
What's the logic used here ?
If `PATH` is not defined then it's necessarily Windows ?

Reply:
An ugly hack that takes advantage of `%PATH%` using semicolons as a delimiter instead of colons. 

Honestly I think there are better options though.

Reply:
I think it's a pretty good differentiation since the $(OS) value for a Cygwin build is just Windows_NT and only Windows uses a semicolon as a PATH separator. However, if you can think of a better way to figure out that a Posix-compatible build on Windows is in progress, feel free to improve that.

Reply:
What about the existing test : `ifneq (,$(filter Windows%,$(OS)))` ? (eventually modified to make it a variable)

Reply:
My tweak of the .exe suffix was unneeded, since even cygwin likes .exe suffixes. The lack of the -fPIC option also seems to work fine, so your change for make install seems good.